account_commission_listing_fee/models/product_product.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2022 Quartile Limited
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
```

Comment:
```suggestion
class ProductProduct(models.Model):
```
I was torn between product.product and product.template, but I think the former (=your design) would be better (more flexible), given that it is not intended to show in the product views.